ios/Classes/components/instant/InstantAdvancedComponent.swift

```
@@ -51,12 +51,14 @@ class InstantAdvancedComponent: BaseInstantComponent, InstantComponentProtocol {
             onError(paymentEventDTO: paymentEventDTO)
         case .action:
             onAction(paymentEventDTO: paymentEventDTO)
+        case .update:
+            return
```

Some components are just silently return, Apple Pay has a dedicated comment for code clarity, just an observation.

---

Good suggestion, will add a comment here as well to enhance code clarity.